src/main/java/de/bonndan/nivio/input/external/gitlab/GitLabConfig.java

```diff
@@ -11,8 +13,16 @@
 
 
 @Configuration
+@EnableConfigurationProperties(GitLabProperties.class)
 public class GitLabConfig {
 
+    private final GitLabProperties gitLabProperties;
+
+    @Autowired
```

Comment:
@Autowired is not necessary in @Configuration classes.

Reply:
Okay, ich habe schon geändert.